nservicebus/gateway/index.md

```diff
@@ -76,6 +76,6 @@ partial: recoverability
 
 ### Version compatibility
 
-The gateway component ensures only forward compatibility for one major version. That means a message sent by the NServiceBus 3.x Gateway can be understood by the NServiceBus 4.x Gateway, and a message sent by the NServiceBus 4.x Gateway can be understood by NServiceBus.Gateway 1.x (the Gateway package targeting NServiceBus 5.x).
+The gateway component ensures only compatibility for one major version. That means a message sent by the NServiceBus 3.x Gateway can be understood by the NServiceBus 4.x Gateway, and a message sent by the NServiceBus 4.x Gateway can be understood by NServiceBus.Gateway 1.x (the Gateway package targeting NServiceBus 5.x).
```

Comment:
Is the major version of the core relevant or the Gateway? I'm confused. Especially because of

> and a message sent by the NServiceBus 4.x Gateway can be understood by NServiceBus.Gateway 1.x (the Gateway package targeting NServiceBus 5.x)

Reply:
I have no idea

Reply:
Me neither. Because I read this and I was thinking if it is the major of the gateway package shouldn't then this text say:

> and a message sent by the NServiceBus 4.x Gateway can be understood by NServiceBus.Gateway 3.x

Reply:
@janovesk ? @andreasohlund ? Do either of you know? Or perhaps @DavidBoike since he added the [original text](https://github.com/Particular/docs.particular.net/commit/72148d846e93fb7cb0ea5907a9b15895fa87e6eb#diff-07dfcd8614218904bd9fe08bf943086c)?